### PR TITLE
Visualize unit tests coverage

### PR DESCRIPTION
**Pull request by tomasperezv**
### What is the goal?

Visualize unit tests coverage in Nevada.

There are 2 modes of coverage visualization:

1. Terminal based via `npm run coverage`
2. Browser based via CircleCI: in each build it will collect the code coverage artifacts (this is also available if you open directly the `coverage/index.html` file in your browser).    

### How is being implemented?

- We are using as a code coverage tool [Istanbul](https://github.com/gotwarlost/istanbul).   
- In order to have compatibility with our Flow + ES6 code we are adding the [nyc](https://github.com/istanbuljs/nyc) wrapper on top of Istanbul.  
- In addition we are excluding some folders and code blocks that we don't want to take into consideration for the coverage reports.

### Reviewers

@jobandtalent/frontend

### CODE REVIEW

File: src/javascript/dom/core.js [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 // Fallback
 let $ = () => {};
+/* istanbul ignore if */
```
Comment (tomasperezv) [inline src/javascript/dom/core.js]: This is the way of excluding certain code branches by annotations.

**Comment by scumah** [comment]
cool :+1:

**Comment by calderon** [comment]
👌

**Comment by alejandroweb** [comment]
👍 Thanks! This is going to be really helpful

**Pull request merged**